src/AppBundle/Model/User/User.php

```diff
@@ -106,7 +106,7 @@ class User implements UserInterface, Serializable
     /**
      * @var string
      *
-     * @ORM\Column(name="activation_key", type="string", nullable=true)
+     * @ORM\Column(name="activation_key", type="string", nullable=true, unique=true, length=255)
```

Comment:
this won't cause conflicts with multiple null values: http://stackoverflow.com/questions/3712222/does-mysql-ignore-null-values-on-unique-constraints